extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,15 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+.amp-story-shopping-product-tag-text {
+  font-family: 'Poppins','Arial', sans-serif !important;
```

Comment:
I don't think it's necessary to specify `font-family` here since it's already specified with the exact same value in the `amp-story-shopping-tag` rule above, so anything within the shopping tag (e.g., elements with the `"amp-story-shopping-product-tag-text"` class) should inherit the rule's `font-family`

Reply:
Yep, you're right, good call.